src/main/java/vazkii/botania/client/integration/nei/recipe/RecipeHandlerManaPool.java

```diff
@@ -86,17 +86,25 @@ public int recipiesPerPage() {
 
 	@Override
 	public void loadTransferRects() {
-		transferRects.add(new RecipeTransferRect(new Rectangle(70, 36, 18, 18), "botania.manaPool"));
+		transferRects.add(new RecipeTransferRect(new Rectangle(70, 21, 18, 18), "botania.manaPool"));
 	}
 
 	@Override
 	public void drawBackground(int recipe) {
 		super.drawBackground(recipe);
+
+		int mana = ((CachedManaPoolRecipe) arecipes.get(recipe)).mana;
+        String manaRequired = Integer.toString(mana);
+        FontRenderer font = Minecraft.getMinecraft().fontRenderer;
+
 		GL11.glEnable(GL11.GL_BLEND);
 		GL11.glColor4f(1.0F, 1.0F, 1.0F, 0.5F);
 		GuiDraw.changeTexture(LibResources.GUI_MANA_INFUSION_OVERLAY);
-		GuiDraw.drawTexturedModalRect(45, 20, 38, 35, 92, 50);
-		HUDHandler.renderManaBar(32, 80, 0x0000FF, 0.75F, ((CachedManaPoolRecipe) arecipes.get(recipe)).mana, TilePool.MAX_MANA / 10);
+
+		GuiDraw.drawTexturedModalRect(45, 5, 38, 35, 92, 50);
+		HUDHandler.renderManaBar(32, 63, 0x0000FF, 0.75F, mana, TilePool.MAX_MANA / 10);
+		font.drawString(manaRequired, 84 - font.getStringWidth(manaRequired) / 2, 71, 0xFF03737F);
```

Comment:
To be eligible, this needs a config option that defaults to not showing the numerical mana value